File: Models/ArtistaMusica.cs
namespace Maia.Models
{
    public class ArtistaMusica
    {
        public int ArtistaId { get; set; }
        public virtual Artista Artista { get; set; }
        public long MusicaId { get; set; }
        public virtual Musica Musica { get; set; }
    }
}

File: Angular/src/app/shared/models/ArtistaMusica.ts
import {Artista} from './Artista';
import {Musica} from './Musica';

export class ArtistaMusica {

  artistaId: number;
  artista: Artista;

  musicaId: number;
  musica: Musica;

  constructor(artistaId?: number, artista?: Artista, musicaId?: number, musica?: Musica) {
    this.artistaId = artistaId;
    this.artista = artista;
    this.musicaId = musicaId;
    this.musica = musica;
  }
}
